.travis.yml

```diff
@@ -51,14 +51,15 @@ jobs:
   include:
     - stage: baseline builds
       os: linux
-      compiler: gcc
-      env: GCOV=gcov CMAKE_FLAGS="$CMAKE_FLAGS -DUSE_GCOV=ON"
-    - os: linux
-      compiler: gcc
+      compiler: clang-4.0
+      # Use Lua so that ASAN can test our embedded Lua support. 8fec4d53d0f6
       env: >
-        FUNCTIONALTEST=functionaltest-lua
+        CLANG_SANITIZER=ASAN_UBSAN
         CMAKE_FLAGS="$CMAKE_FLAGS -DPREFER_LUA=ON"
-        DEPS_CMAKE_FLAGS="$DEPS_CMAKE_FLAGS -DUSE_BUNDLED_LUAJIT=OFF"
+        ASAN_SYMBOLIZE=asan_symbolize-4.0
+    - os: linux
+      compiler: gcc
+      env: GCOV=gcov CMAKE_FLAGS="$CMAKE_FLAGS -DUSE_GCOV=ON"
```

Comment:
gcov should not be in the early builds.

Reply:
Why not?

Reply:
why?

Reply:
Since it provides one of the PR statuses, and should not be skipped with flaky builds.
But moved it back now.